atom.xml

```diff
@@ -5824,5 +5824,269 @@
 		<summary>Automatically switch css html background color.</summary>
 	</entry>
 
+	<entry>
```

Comment:
Like the repo readme says, PRs adding new libraries should not change atom.xml or data.min.js. You should only modify data.js. You should change your branch to no longer modify these files, or create a new PR without this diff.